.storybook/preview.js

```diff
@@ -37,34 +62,28 @@ addDecorator(
     },
     {
       title: 'New Design Language',
-      components: [AppProvider],
+      components: [AppProviderWithKnobs],
       params: [
         {
           name: 'Disabled',
           default: true,
-          props: {i18n: enTranslations},
+          props: {newDesignLanguage: false},
         },
         {
           name: 'Enabled - Light Mode',
-          props: {
-            i18n: enTranslations,
-            features: {newDesignLanguage: true},
-            theme: {colorScheme: 'light'},
-          },
+          props: {newDesignLanguage: true, colorScheme: 'light'},
         },
         {
           name: 'Enabled - Dark Mode',
-          props: {
-            i18n: enTranslations,
-            features: {newDesignLanguage: true},
-            theme: {colorScheme: 'dark'},
-          },
+          props: {newDesignLanguage: true, colorScheme: 'dark'},
         },
       ],
     },
   ]),
 );
 
+addDecorator(withKnobs);
```

Comment:
Referencing ` '@storybook/addon-knobs'` instead of ` '@storybook/addon-knobs/register'` in main.js means that we don't need this line anymore - as it is already implictly added by activating the addon by its root instead of the `/register` path

(don't forget to remove the import of withKnobs)

Reply:
Looks like this is needed